src/main/java/net/mindengine/galen/utils/GalenUtils.java

```diff
@@ -380,16 +380,21 @@ public static InputStream findFileOrResourceAsStream(String filePath) throws Fil
             return new FileInputStream(file);
         }
         else {
-            if (!filePath.startsWith("/")) {
-                filePath = "/" + filePath;
+            if (!filePath.startsWith(File.separator)) {
```

Comment:
I think this is not going to work in case the specs are referenced in project as resourced instead of direct file path. It was a long time ago but I remember that this part of a code was important for this specific case